test/Microsoft.ML.Benchmarks/KMeansAndLogisticRegressionBench.cs

```diff
@@ -21,7 +21,7 @@ public class KMeansAndLogisticRegressionBench
         [Benchmark]
         public ParameterMixingCalibratedPredictor TrainKMeansAndLR()
         {
-            using (var env = new ConsoleEnvironment(seed: 1))
+            using (var env = new ConsoleEnvironment(seed: 1, verbose: false, sensitivity: MessageSensitivity.None, outWriter: EmptyWriter.Instance))
```

Comment:
Does this hide any actual errors which may occur?

Reply:
How about just using `MLContext` instead? It is what we tell our users to use.

Using a `ConsoleEnvironment`, and then telling it to write to `/dev/null` makes no sense IMO.

Reply:
agree

---
In reply to: [230093244](https://github.com/dotnet/machinelearning/pull/1476#discussion_r230093244) [](ancestors = 230093244)